buildkite/src/Command/DeployNightly.dhall

```diff
@@ -0,0 +1,31 @@
+let Prelude = ../External/Prelude.dhall
+
+let Command = ./Base.dhall
+let Size = ./Size.dhall
+
+let Cmd = ../Lib/Cmds.dhall
+-- do we need this?
```

Comment:
The dependency? Only if we don't have an exact image to use. Based on the current tagging system, tough to make use of a static entry that is reliably up-to-date. We could add a latest or nightly etc tag to the builds tho to get around this.